packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/classes_table_single.dart

```diff
@@ -133,15 +116,12 @@ class SnapshotStatsTable extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    final data = item.statsToShow.classStats;
-    return FlatTable<HeapClassStatistics>(
-      keyFactory: (e) => Key(e.heapClass.fullName),
-      data: data,
-      dataKey: 'snapshot-stats-$dataId',
+    return FlatTable<SingleClassStats>(
       columns: _columns,
-      selectionNotifier: item.selectedClassStats,
-      onItemSelected: (classStats) =>
-          controller.selectedClassName.value = classStats?.heapClass.fullName,
+      data: classes.classes,
+      dataKey: 'ClassesTableSingle',
```

Comment:
Just to make sure this is intended, by changing from 'snapshot-stats-$dataId' to 'ClassesTableSingle', each snapshot's sort column and direction will not be preserved, but rather a change to sort column or direction will be applied to the table for every snapshot.

Reply:
Yes, this is how Finder behaves and I think we should do the same here. I doubt anybody will ever want different sorting for different snapshots.